enterprise/frontend/src/embedding-sdk/components/private/SdkGlobalStylesWrapper.unit.spec.tsx

```diff
@@ -1,61 +0,0 @@
-import { renderWithProviders, screen } from "__support__/ui";
```

Comment:
I'll add the tests back if you approve of this "strategy", Ideally i'd put them on MetabaseProvider directly so that they're abstracted from lower level implementation details